### ISSUE TITLE
Request for scripts of fine-tuning

### ISSUE THREAD

**Issue by MT010104** [hendrycks/apps#18]
Hi, thanks for the amazing work! I really appreciate that you released the dataset, but now I wanna apply it to other models downloaded from Hugging Face. I wonder if I can get the scripts of fine-tuning?

**Comment by xksteven** [comment]
Are these the instructions you're looking for? 
https://github.com/hendrycks/apps/blob/main/train/README.md

**Comment by MT010104** [comment]
Thanks for the code provided, but now I have a new problem. There is a great difference betweem the code generated by the finetuned model ("codet5-small") and the content in solutions.json. The training parameters are hardly modified at all and the loss function curve is also normal. Here's the generated code of the second problem in APPS/test/0001. 
![172 19 135 188 (yanming) 2022_10_11 14_31_27(1)](https://user-images.githubusercontent.com/65756145/195013666-f9d93436-59af-4aae-9d53-90823f63dbf0.png)

**Comment by xksteven** [comment]
I am closing this issue as there could be many many reasons as to why you're model is not working.  Those questions are probably best addressed on discord or stack overflow though.  Please do feel free to open a new issue if you have any trouble reproducing our results. 

We also provide the weights to several fine-tuned models in the README link pasted here as well: https://drive.google.com/file/d/1XW1Od9L-5l9zXl1HUCyER5pS9zQTbIvU/view?usp=sharing

Best of luck debugging
